ansible/roles/operator-pipeline/templates/openshift/tasks/upload-pipeline-logs.yml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: upload-pipeline-logs
+spec:
+  params:
+    - name: pyxis_api_key_secret
+      default: pyxis-api-secret
+    - name: pyxis_url
+      default: https://catalog.redhat.com/api/containers/
+    - name: md5sum
+    - name: cert_project_id
+    - name: bundle_version
+    - name: package_name
+    - name: pipeline_name
+  results:
+    - name: pipeline_log_url
+  steps:
+    - name: get-pipeline-logs
+      image: registry.redhat.io/openshift-pipelines/pipelines-cli-tkn-rhel8:0.19.0-2
+      script: |
+        #! /usr/bin/env bash
+        set -x
```

Comment:
Do we need the `-e` switch as well to ensure the script errors out for non-zero exit codes?

Reply:
added